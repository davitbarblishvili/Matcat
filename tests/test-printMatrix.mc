func main() int {
    matrix m; 
    m = [[1,2,3],[4,5,6]];
    printm(m);

    return 0;
}