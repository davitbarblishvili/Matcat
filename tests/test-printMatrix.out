(
[1 2 3 ]
[4 5 6 ]
[10 20 30 ]
[100 255 560 ]
)